doc/user-guide/Bootstrap-Scripts.md

```diff
@@ -20,22 +20,20 @@ Environment variables, available from scripts:
 - `MIM_DIR` - MongooseIM release installation directory.
 
 
-# Templating bootstrap script
+## Templating bootstrap script
 
 The script `bootstrap20-template.escript` renders files from the `templates/` directory and writes
 result files into the `etc/` directory. If you need the result files in a separate directory,
 create another script `bootstrap30-template.sh`, that moves files into a proper location.
 
-
 The `etc/templates.ini` file contains default template variables.
 
-
 A template config example:
 
-```erlang
+```toml
 [options]
-demo_session_lifetime = 600
-demo_tls_versions = 'tlsv1.2', 'tlsv1.3'
+  demo_session_lifetime = 600
+  demo_tls_versions = 'tlsv1.2', 'tlsv1.3'
```

Comment:
Wait... This is not toml, but actually ini. It's about bootstrap scripts, and not MIM configuration.

Reply:
will fix

Reply:
fixed, but now imho it looks worse in browser 🙄 anyway, let's stick to the correct type